pkg/resources/jobs/initializer.go

```diff
@@ -18,7 +18,7 @@ func NewInitializerJob(k6 *v1alpha1.K6, argLine string) (*batchv1.Job, error) {
 	}
 
 	var (
-		image                        = "ghcr.io/grafana/operator:latest-runner"
+		image                        = "ghcr.io/grafana/k6-operator:latest-runner"
```

Comment:
This change, along with similar ones in `*.go` files, breaks the operator until there's `latest` image is built. E2E test in Github Actions also is broken without it. So I've built a `new-pkgname` build from this branch manually, to have a `latest` image in `k6-operator` package. This way nobody should be affected by the change.